### ISSUE TITLE
Delivering source status fails for OCP

### ISSUE THREAD

**Issue by dccurtis** [project-koku/koku#2055]
This problem has only been seen in CI and has not been reproduced locally.

Need to investigate why `ProviderAccessor` is failing to find the correct source type.  I did verify that the database had the correct source type and all other source creation and destruction operations were not impacted by this.

```
[2020-04-14 17:44:39,349] INFO None Delivering source status for Source ID: 6582
[2020-04-14 17:44:39,421] ERROR None  is not a valid provider
[2020-04-14 17:44:39,425] INFO None {'method': 'POST', 'path': '/api/cost-management/v1/source-status', 'status': 204, 'request_id': None, 'account': None, 'username': None, 'is_admin': False}
Exception in thread Thread-591:
Traceback (most recent call last):
  File "/opt/rh/rh-python36/root/usr/lib64/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/opt/app-root/lib/python3.6/site-packages/sentry_sdk/integrations/threading.py", line 69, in run
    reraise(*_capture_exception())
  File "/opt/app-root/lib/python3.6/site-packages/sentry_sdk/_compat.py", line 57, in reraise
    raise value
  File "/opt/app-root/lib/python3.6/site-packages/sentry_sdk/integrations/threading.py", line 67, in run
    return old_run_func(self, *a, **kw)
  File "/opt/rh/rh-python36/root/usr/lib64/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/app-root/src/koku/sources/api/source_status.py", line 76, in push_status
    status_msg = self.status()
  File "/opt/app-root/src/koku/sources/api/source_status.py", line 70, in status
    availability_status = interface.availability_status(source_authentication, source_billing_source)
  File "/opt/app-root/src/koku/providers/provider_access.py", line 132, in availability_status
    self.cost_usage_source_ready(credential, source_name)
  File "/opt/app-root/src/koku/providers/provider_access.py", line 106, in cost_usage_source_ready
    return self.service.cost_usage_source_is_reachable(credential, source_name)
AttributeError: 'NoneType' object has no attribute 'cost_usage_source_is_reachable'
[2020-04-14 17:44:39,657] INFO None Adding operation create for Doug OCP test to process queue (size: 0)
```

**Comment by dccurtis** [comment]
Already fixed with referenced PR

**Comment by dccurtis** [comment]
Problem was fixed with #2057
